src/components/VersionFileViewer/index.spec.tsx

```diff
@@ -143,16 +144,18 @@ describe(__filename, () => {
   it('renders a FileTree component', () => {
     const onSelectFile = jest.fn();
     const { version } = getInternalVersionAndFile();
+    const comparedToVersionId = 11;
 
     const root = renderPanel(
-      { onSelectFile, version },
+      { onSelectFile, version, comparedToVersionId },
       PanelAttribs.mainSidePanel,
     );
 
     const tree = root.find(FileTree);
     expect(tree).toHaveLength(1);
     expect(tree).toHaveProp('onSelect', onSelectFile);
     expect(tree).toHaveProp('versionId', version.id);
+    expect(tree).toHaveProp('comparedToVersionId', comparedToVersionId);
```

Comment:
The `KeyboardShortcuts` test should get a similar assertion to make sure `comparedToVersionId` is passed all the way through.